### ISSUE TITLE
Mark items which have been taken out by library staff as temporarily unavailable

### ISSUE THREAD

**Issue by alexwlchan** [wellcomecollection/platform#5632]
Wellcome staff are able to check out books from the library, but these still appear as "Open shelves / available" in the online catalogue (and OPAC).

This causes confusion for LE&E staff when they go to look for a book on the shelves, not realising it's been checked out. Neither they nor the reader can find the book, and it's not until they look in Sierra that they see what the issue is.

We should mark such items as "temporarily unavailable, please contact the library" – I think we can do this by looking at the presence of a DUE DATE field in the Sierra record.

**Comment by alexwlchan** [comment]
Here's a message we put on some items (https://wellcomecollection.org/works/qhzks7b2)

<img width="786" alt="Screenshot 2022-11-22 at 09 46 39" src="https://user-images.githubusercontent.com/301220/203281453-60722bd4-c620-423a-8d8e-fdf5fa88c503.png">

**Comment by alexwlchan** [comment]
Also, a quick scan shows this affects ~1400 items currently (some of which already have the message, like the example above).

**Comment by pollecuttn** [comment]
@alexwlchan Should this have been fixed with the new pipeline?

**Comment by alexwlchan** [comment]
Right, so the catalogue API is returning the right data about this item:

```
          "accessConditions": [
            {
              "method": {
                "id": "open-shelves",
                "label": "Open shelves",
                "type": "AccessMethod"
              },
              "status": {
                "id": "temporarily-unavailable",
                "label": "Temporarily unavailable",
                "type": "AccessStatus"
              },
              "note": "Item is in use by another reader. Please ask at Library Enquiry Desk.",
              "type": "AccessCondition"
            }
          ],
```

but the items API has lost the access note:

```
                    "accessConditions": [
                        {
                            "method": {
                                "id": "not-requestable",
                                "label": "Not requestable",
                                "type": "AccessMethod"
                            },
                            "type": "AccessCondition"
                        }
                    ],
```

Which is why it's not appearing on the page. Hmm.

**Comment by pollecuttn** [comment]
25/5/23 Checked out https://wellcomecollection.org/works/g2773269 to myself at 8.51 today.

**Comment by alexwlchan** [comment]
Natalie and I have done some testing today.

After an item is checked out (or in), it takes a few minutes for that information to propagate through the catalogue pipeline before the item is updated on the site. The pipeline knows about due dates; the items API doesn't.

I think that's good enough for this ticket – there's already an inherent latency in this process, e.g. between somebody removing a book from the shelf and taking it to the desk to be checked out, when Sierra might think a book is on the shelves but nobody else can find it. If it becomes a big problem we can revisit this, but our effort is probably better spent speeding up the catalogue pipeline.
